packages/odyssey-storybook/src/utilities/Text/TextElements.stories.tsx

```diff
@@ -17,7 +17,7 @@ import type { TextProps } from "@okta/odyssey-react";
 import { Text as Source } from "../../../../odyssey-react/src";
 
 export default {
-  title: `Components/Text/Elements`,
+  title: `Utilities/Text/Elements`,
```

Comment:
I'm not sure I agree Text and ScreenReaderText should be listed under utilities and not components? The distinction seems artificial to me - developers can use both of these components (and really Box too) directly if they want to?

Reply:
I'm definitely open here. In the past, we've discussed "component" being problematic generally. 

On the Design and Product side, "component" is more equitable to "atom" - a discrete piece of rendered UI with a distinct _product_ purpose.

In React world, it has a technical meaning which makes calling these "Utilities" incorrect, 100% agreed there.

We've pushed a few different approaches to the language in the past (mainly Atomic), but we've never been able to get folks at large to agree. :(